### PR TITLE
Update io-cdc-canal.md

### PR DESCRIPTION
**Pull request by tomix1337**
Fixed a couple of things:
Pulsar 2.3.0 connectors gives a 404 error.
pulsar-admin does not support a "sources" command, but a "source".

*(If this PR fixes a github issue, please add `Fixes #<xyz>`.)*

Fixes #<xyz>

*(or if this PR is one task of a github issue, please add `Master Issue: #<xyz>` to link to the master issue.)*

Master Issue: #<xyz>

### Motivation
Readme update.

### Modifications

Inside io-cdc-canal.md, 
Pulsar 2.3.0 connectors gives a 404 error.
pulsar-admin does not support a "sources" command, but a "source".

### Verifying this change

- [x] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

[x] This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

### Does this pull request potentially affect one of the following parts:

*If `yes` was chosen, please highlight the changes*

  - Dependencies (does it add or upgrade a dependency): (no)
  - The public API: (no)
  - The schema: (no)
  - The default values of configurations: (no)
  - The wire protocol: (no)
  - The rest endpoints: (no)
  - The admin cli options: (no)
  - Anything that affects deployment: (no)

### Documentation

  - Does this pull request introduce a new feature? (no)
  - If yes, how is the feature documented? (not applicable)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

File: site2/docs/io-cdc-canal.md [hunk 1]
```diff
@@ -163,9 +163,9 @@ Here is an example of storing MySQL data using the configuration file as above.
     
     ```bash
     $ docker exec -it pulsar-standalone /bin/bash
-    $ wget http://apache.01link.hk/pulsar/pulsar-2.3.0/connectors/pulsar-io-canal-2.3.0.nar -P connectors
-    $ ./bin/pulsar-admin sources localrun \
-    --archive ./connectors/pulsar-io-canal-2.3.0.nar \
+    $ wget http://apache.01link.hk/pulsar/pulsar-2.4.1/connectors/pulsar-io-canal-2.4.1.nar -P connectors
+    $ ./bin/pulsar-admin source localrun \
+    --archive ./connectors/pulsar-io-canal-2.4.1.nar \
     --classname org.apache.pulsar.io.canal.CanalStringSource \
```
Comment (tuteng) [inline site2/docs/io-cdc-canal.md]: You may also need to change this file https://github.com/apache/pulsar/blob/master/site2/website/versioned_docs/version-2.4.1/io-cdc-canal.md and replace the version number of 2.3.0 with 2.4.1.

**Comment by tuteng** [comment]
run java8 tests
run cpp tests
run integration tests

**Comment by tuteng** [comment]
run integration tests

**Comment by tomix1337** [comment]
@tuteng can we merge? Everything should be ready.

**Pull request merged**